### PR TITLE
Fix release script tagging for cli image

### PR DESCRIPTION
**Pull request by arroyc**
as title said

### CODE REVIEW

File: vsts/scripts/tagBuildImagesForRelease.sh [hunk 1]
```diff
@@ -11,16 +11,11 @@ declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && pwd )
 source $REPO_DIR/build/__variables.sh
 
 outPmeFileMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/oryxprodmcr-build-images-mcr.txt"
-outNonPmeFileMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/oryxmcr-build-images-mcr.txt"
```
Comment (qianz2) [inline vsts/scripts/tagBuildImagesForRelease.sh:14]: There still have lots of lines involving `NonPme` were not removed from these four scripts. I suggest you search the keyword in these files and make sure the PR's removed all lines.

Comment (arroyc) [reply vsts/scripts/tagBuildImagesForRelease.sh:14]: thanks I will remove this and anything that are remaining as well

File: vsts/scripts/tagBuildImagesForRelease.sh [hunk 2]
```diff
@@ -11,21 +11,15 @@ declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && pwd )
...
 
-if [ -f "$outNonPmeFileMCR" ]; then
-    rm $outNonPmeFileMCR
-fi
-
 function tagBuildImage() {
     local devRegistryImageName="$1"
     local prodRegistryLatestTagName="$2"
     local prodRegistrySpecificTagName="$3"
-    local prodNonPmeRegistryRepoName="oryxmcr.azurecr.io/public/oryx/build"
```
Comment (qianz2) [inline vsts/scripts/tagBuildImagesForRelease.sh:28]: remove lines: 41-44, 52-55, 73-75

File: vsts/scripts/tagCliImagesForRelease.sh [hunk 3]
```diff
@@ -55,7 +49,4 @@ fi
 
 echo "printing pme tags from $outPmeFile"
 cat $outPmeFile
-echo -------------------------------------------------------------------------------
```
Comment (qianz2) [inline vsts/scripts/tagCliImagesForRelease.sh:58]: remove lines: 32-34, 42-44, 52-53, 59-60

File: vsts/scripts/tagRunTimeImagesForRelease.sh [hunk 4]
```diff
@@ -10,18 +10,13 @@ acrNonPmeProdRepo="oryxmcr"
 acrPmeProdRepo="oryxprodmcr"
 
 sourceBranchName=$BUILD_SOURCEBRANCHNAME
-outFileNonPmeMCR="$BUILD_ARTIFACTSTAGINGDIRECTORY/drop/images/$acrNonPmeProdRepo-runtime-images-mcr.txt"
```
Comment (qianz2) [inline vsts/scripts/tagRunTimeImagesForRelease.sh:13]: remove lines: 9, 49, 51-52

**Review by qianz2 — APPROVED** [review]

**Pull request merged**